Integrations/Snowflake/Snowflake.yml

```diff
@@ -0,0 +1,179 @@
+category: Database
+commonfields:
+  id: Snowflake
+  version: -1
+configuration:
```

Comment:
We typically use that param when using the requests library. Because this integration uses snowflake's python module and not a REST API, I don't see how it would be used in this context.